apps/_compability_app/common/components/CustomAvatar.jsx

```diff
@@ -0,0 +1,35 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import styled from "@emotion/styled";
+import { Avatar } from "antd"
+import React from "react";
+import { useIntl } from "react-intl";
```

Comment:
(Optional) I think, It's better to import it from @core. Because it's help to use exactly the same Intl version as for _app. Ours help to avoid to use multiple versions of Intl.